src/api/appData.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto2";
+
+package api;
+
+message AppData {
+        optional bool motion = 1;
+	optional uint32 water = 2;
+	optional float temperature_celcius = 3;
+	optional float temperature_fahrenheit = 4;
+	optional float humidity = 5;
```

Comment:
This is not what the app is going to send back. Please consult @FokkeZB for an example